code/game/gamemodes/storyteller.dm

```diff
@@ -153,6 +154,9 @@ GLOBAL_VAR_INIT(chaos_level, 1) //Works as global multiplier for all storyteller
 
 	last_tick = world.time
 	next_tick = last_tick + tick_interval
+	if(REALTIMEOFDAY > last_chaos_hour + 1 HOUR)
+		GLOB.chaos_level += 1
+		last_chaos_hour += 1 HOUR
```

Comment:
Stealth change and reimplementation of the  previously reverted PR..

Reply:
... I merged your revert, that wasn't supposed to be a thing.

Reply:
why didn't your revert remove it?

Reply:
it "removed the feature", right?
why did you let the var stay?

Reply:
wait no
this was just merging before your PR was merged, I think.
will fix.